account-kit/smart-contracts/src/coinbase/tests/coinbase-account.test.ts

```diff
@@ -0,0 +1,144 @@
+import {
+  toCoinbaseSmartAccount,
+  createBundlerClient,
+  bundlerActions,
+} from "viem/account-abstraction";
+import { generatePrivateKey, privateKeyToAccount } from "viem/accounts";
+import { getBlockNumber, setBalance, getBalance } from "viem/actions";
+import { parseEther, custom, publicActions } from "viem";
+import { describe, it, expect, beforeAll } from "vitest";
+import { local070Instance } from "../../../../../.vitest/src/instances.js";
+
+describe("Viem AA - Coinbase Smart Account", () => {
+  let client: ReturnType<typeof local070Instance.getClient>;
+
+  beforeAll(async () => {
+    client = local070Instance.getClient();
+    const blockNumber = await getBlockNumber(client);
+    expect(blockNumber).toBeGreaterThan(0n);
+  }, 30_000);
+
+  it("should create a Coinbase smart account successfully", async () => {
+    const owner = privateKeyToAccount(generatePrivateKey());
+
+    const account = await toCoinbaseSmartAccount({
+      client,
+      owners: [owner],
+    });
+
+    expect(account).toBeDefined();
+    expect(account.address).toMatch(/^0x[a-fA-F0-9]{40}$/);
+    expect(account.type).toBe("smart");
+  });
+
+  it("should send a user operation (ETH transfer)", async () => {
+    const owner = privateKeyToAccount(generatePrivateKey());
+
+    const account = await toCoinbaseSmartAccount({
+      client,
+      owners: [owner],
+    });
+
+    const bundlerClient = createBundlerClient({
+      account,
+      chain: local070Instance.chain,
+      transport: custom(client),
+    }).extend(bundlerActions);
```

Comment:
Interesting, does the test not work if we don't include `.extend(bundlerActions)`? I would expect the bundler client to include these already.